backend_uml.go

```diff
@@ -50,11 +50,14 @@ func (b umlBackend) KernelRelease() (string, error) {
 
 func (b umlBackend) KernelPath() (string, error) {
 	// find the UML binary
-	kernelPath, err := exec.LookPath("linux.uml")
-	if err != nil {
-		return "", fmt.Errorf("user-mode-linux not installed")
+	kernelBinaries := []string{"linux.uml", "vmlinux"}
```

Comment:
Cannot see anything custom in the Arch recipe. On the other hand the [Debian one](https://salsa.debian.org/uml-team/user-mode-linux/-/blob/master/debian/rules#L114) is responsible for the different name and module location.

So I would recommend tweaking this and the previous patch to:
 - search the standard location first
 - tweak the commit message (+ potentially add inline comment) that Debian moves/renames things around